.github/workflows/docker.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Publish Docker Image
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish to Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
```

Comment:
Perhaps we should pin this to a tagged version to avoid breakage? I don't know if actions support semantic versioning.

Latest tag seems to be `3.01`: https://github.com/elgohr/Publish-Docker-Github-Action/tree/3.01

Reply:
True. Done 👍